vermouth/processors/do_links.py

```diff
@@ -98,7 +98,7 @@ def _interpret_order(order):
     return order_type, order_value
 
 
-def match_order(order1, resid1, order2, resid2):
+def match_order(order1, resid1, chain1, order2, resid2, chain2):
```

Comment:
A different thing, that should be fixed: this changes the public vermouth API, and that breaks polyply. Could you make the chain arguments optional?

Reply:
That requires changing the order to completely illogical. But it's doable.

Reply:
Agreed, but fugly API is preferable to broken API I'd say.